test/ports-map.js

```diff
@@ -46,7 +46,7 @@ const portsList = {
   Iframe: 1,
 };
 
-let startPort = 8079;
+let startPort = 8089;
```

Comment:
Why did you change this? I thought it was only necessary to change the number of cli from 2 to 10.

Reply:
@hiroppy There are other tests that use the default ports near 8080, such as: https://github.com/webpack/webpack-dev-server/blob/master/test/server/utils/findPort.test.js. So I think we should provide some extra room to make sure those ports are available.